history/characters/200000_farraki.txt

```diff
@@ -8632,3 +8634,57 @@
 	635.8.7={ birth=yes }
 	718.9.14={ death=yes }
 }
+ruuzlu = { # right-hand man and friend of Ukorz Sandscalp
+	name = Ruuzlu
```

Comment:
The name need localization.
> [16:15:34][characterhistory.cpp:698]: Missing loc for name 'Ruuzlu' for character 'ruuzlu'

Reply:
Added in 193de1154b6fba4cf35e31de3e5d4fa26faa909a.